zokrates_cli/src/bin.rs

```diff
@@ -448,7 +448,7 @@ fn cli() -> Result<(), String> {
     let matches = App::new("ZoKrates")
     .setting(AppSettings::SubcommandRequiredElseHelp)
     .version(env!("CARGO_PKG_VERSION"))
-    .author("Jacob Eberhardt, Thibaut Schaeffer, Stefan Deml")
+    .author("Jacob Eberhardt, Thibaut Schaeffer, Stefan Deml, Darko Masesic")
```

Comment:
Just a quick note, its "Darko Macesic" :D

Reply:
🤦 🤦  and I did check....................
```suggestion
    .author("Jacob Eberhardt, Thibaut Schaeffer, Stefan Deml, Darko Macesic")
```

Reply:
That's a 🍺